roles/windows-executor/tasks/install_windows_updates.yml

```diff
@@ -1,10 +1,10 @@
 # !file: roles/windows-executor/tasks/install_windows_updates.yml
 
-- name: Install all security, critical, and rollup updates
-  win_updates:
-    category_names:
-      - SecurityUpdates
-      - CriticalUpdates
-      - UpdateRollups
+- name: Install all updates and reboot as many times as needed
+  ansible.windows.win_updates:
```

Comment:
is it safe to assume this module has always been available, and we'd just been using the older deprecated `win_updates` module until now?

Reply:
@bjohnso5 we were using this same module however I wanted to be more explicit so there was no confusion with the community version.